mock/economicsDataMock.go

```diff
@@ -1,13 +1,15 @@
 package mock
 
 import (
-	"fmt"
 	"math/big"
 
 	"github.com/ElrondNetwork/elrond-go-core/core"
 	coreData "github.com/ElrondNetwork/elrond-go-core/data"
+	logger "github.com/ElrondNetwork/elrond-go-logger"
 )
 
+var log = logger.GetOrCreate("mock/economicsData")
```

Comment:
`logger.GetOrCreate("mock")` - we declare the package's path in these situations